fe/fe-core/src/main/java/com/starrocks/load/DppConfig.java

```diff
@@ -56,7 +56,7 @@
     private static final String APPLICATIONS_PATH = "applications";
     private static final String OUTPUT_PATH = "output";
 
-    public static final String STARROCKS_PATH = "hadoop_palo_path";
+    public static final String STARROCKS_PATH = "hadoop_starrocks_path";
```

Comment:
this config key should not changed easily.
This should keep compatible with old version

Reply:
These two files are useless and can be deleted.